File: javascript/examples/get_filing_methods_schemas.js
const baseRequest = require('../base_request.js').baseRequest

const path = require('path');
process.chdir(path.resolve(__dirname, '../../'));
require('dotenv').config();

// GET /filing-methods/schemas
const COMPANY_ID	= process.env.COMPANY_ID;
const METHOD_ID 	= process.env.FILING_METHOD_ID;

params = {
  company_id: COMPANY_ID,
  filing_method_id: METHOD_ID
}

token = baseRequest.request.token({ path: '/filing-methods/schemas' });

baseRequest.request.get({ path: '/filing-methods/schemas', token: token, queryParams: params });

File: ruby/examples/get_filing_methods_schemas.rb
require_relative '../request.rb'

# Example of GET /filing-methods/schemas

COMPANY_ID        = ENV['COMPANY_ID']
FILING_METHOD_ID  = ENV['FILING_METHOD_ID']

params = {
  company_id: COMPANY_ID,
  filing_method_id: FILING_METHOD_ID
}

BaseRequestRoute.request(:get, '/filing-methods/schemas', query_params: params)
